elephant/statistics.py

```diff
@@ -799,23 +809,416 @@ def complexity_pdf(spiketrains, binsize):
            pp. 96-114, Springer Berlin Heidelberg, 2008.
 
     """
-    # Computing the population histogram with parameter binary=True to clip the
-    # spike trains before summing
-    pophist = time_histogram(spiketrains, binsize, binary=True)
+    warnings.warn("complexity_pdf is deprecated in favor of the complexity "
+                  "class which has a pdf attribute. complexity_pdf will be "
+                  "removed in the next Elephant release.", DeprecationWarning)
+
+    complexity_obj = complexity(spiketrains, bin_size=binsize)
+
+    return complexity_obj.pdf
+
+
+class complexity:
```

Comment:
Is there a particular reason why you called the class name lowercase? Python standards tell using CamelCase naming for classes. Same for `Synchrotool` (or `SynchroTool`) class.